.github/workflows/CI.yml

```
@@ -37,7 +37,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [10.x, 12.x]
+        node-version: [12.x, 14.x]
```

Let's test on LTS too:

```suggestion
        node-version: [12.x, 14.x, 16.x]
```